api/v4/source/audit_logging.yaml

```diff
@@ -0,0 +1,68 @@
+  /api/v4/audit_logs/certificate:
+    post:
+      tags:
+        - audit_logs
+      summary: Upload audit log certificate
+      description: |
+        Upload the certificate to be used for TLS verification with the audit log service.
+
+        ##### Permissions
+        Must have `sysconsole_write_experimental_features` permission.
+
+        __Minimum server version__: 9.5
```

Comment:
I guess this should be updated to 10.8 or 10.9?

Reply:
```suggestion
        __Minimum server version__: 10.9
```